tests/SearchClient.spec.ts

```diff
@@ -2,8 +2,22 @@ import {suite, test, timeout} from "mocha-typescript";
 import SearchClient from "../src/SearchClient";
 import {expect} from "chai";
 
+const nock = require('nock');
+
 @suite("SearchClientSpec", timeout(100000))
 class SearchClientSpec {
+  async before() {
+    let appId: string = process.env["appId"] as string;
+    nock(`https://${appId}-fast.searchtap.net/v2`)
```

Comment:
@asinghall - we should remove the dependency on appId. Hardcode an app-id like `sample`